docs/index.md

```diff
@@ -1,30 +1,59 @@
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
-for deploying and managing the [Chrony](https://chrony-project.org) NTP server 
+# Chrony Operator
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+for deploying and managing the [Chrony](https://chrony-project.org) NTP server
 in your systems.
 
-This charm simplifies the configuration and maintenance of `chrony` across a 
-range of environments, enabling basic time provision and synchronization with 
+This charm simplifies the configuration and maintenance of `chrony` across a
```

Comment:
Nitpick: `chrony` --> Chrony 

Unless there's a reason why you used code formatting here?

Reply:
Updated to Chrony, thanks!